packages/roosterjs-editor-core/lib/corePlugins/utils/InlineEntityHandlers/inlineEntityOnPluginEvent.ts

```diff
@@ -0,0 +1,56 @@
+import { DelimiterClasses, EntityOperation, PluginEventType } from 'roosterjs-editor-types';
+import { getDelimiterFromElement, isBlockElement, safeInstanceOf } from 'roosterjs-editor-dom';
+import type { Entity, PluginEvent } from 'roosterjs-editor-types';
+
+const DELIMITER_SELECTOR =
+    '.' + DelimiterClasses.DELIMITER_AFTER + ',.' + DelimiterClasses.DELIMITER_BEFORE;
+
+export function inlineEntityOnPluginEvent(event: PluginEvent) {
+    switch (event.eventType) {
+        case PluginEventType.ExtractContentWithDom:
+        case PluginEventType.BeforeCutCopy:
+            event.clonedRoot.querySelectorAll(DELIMITER_SELECTOR).forEach(node => {
+                node.parentElement?.removeChild(node);
```

Comment:
Since we remove delimiter when copy or save, we also need to add them back when load content.

Reply:
Good point. Will handle it in a future step.